README.md

```diff
@@ -148,44 +134,20 @@ Download a CoreOS image into `$PWD/.cache/images`.
 
 `cork download-image --platform=qemu`
 
-## building CoreOS on CoreOS
-
-Here is an example script that will download and build a CoreOS image using
-cork and the SDK. It is assumed that it is run on an existing CoreOS instance.
-
-The resulting QEMU images will be in
-`$HOME/coreos/src/build/images/amd64-usr/latest/`.
+### Building CoreOS with cork
 
-```sh
-# setup env vars
-export PATH=$HOME/bin:$PATH
-export V=master
-export S=/mnt/host/source/src/scripts
-export B=amd64-usr
+See [Modifying CoreOS](https://coreos.com/os/docs/latest/sdk-modifying-coreos.html) for
+an example of using cork to build a CoreOS image.
 
-# build mantle commands, including cork
-cd $HOME
-git clone https://github.com/coreos/mantle.git
-cd mantle
-docker run --rm -v "$PWD":/usr/src/myapp -w /usr/src/myapp golang ./build
-mkdir -p $HOME/bin/
-cp bin/* $HOME/bin/
-
-mkdir $HOME/coreos
-cd $HOME/coreos
+## plume
 
-# download CoreOS SDK, build packages and image
-cork create --replace --sdk-version=${V} --verbose
-cork enter -- ${S}/build_packages --board=${B}
-cork enter -- ${S}/build_image --board=${B} dev prod
+CoreOS release utility
 
-# optionally you can run the kola tests on the built image
-cork enter -- sudo kola run -v
+### plume index
 
-# build dev and prod QEMU images
-cork enter -- ${S}/image_to_vm.sh --board=${B}
-cork enter -- ${S}/image_to_vm.sh --board=${B} --prod_image
-```
+Generate and upload index.html objects to turn a Google Cloud Storage
+bucket into a publicly browsable file tree. Useful if you want something
+like Apache's directory index for your software download repository.
 
 ## TODO
```

Comment:
I tend to think TODOs belong in the issue tracker anyway, so I'd be +1 to dropping this.